services/multilang-support/src/translate.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+from flask import Flask, request, jsonify
+from .utils import translate_helsinki, LOGGER
+import json
+import jsonschema
+
+app = Flask(__name__)
+
+# Load schema
+"""
+(Check the Dockerfile)
+Here we send the app to / directory, along with the schema file.
+Hence, we can load the schema file as below. (relative path)
+"""
+
+# uncomment this line to test the server locally (without Docker)
+# with open("../../schemas/services/translation.schema.json", "r") as f:
+with open("translation.schema.json", "r") as f:
+    TRANSLATION_SCHEMA = json.load(f)
+
+# Validate request
+
+
+def validate_request(request):
+    """
+    Validate incoming request:
+    - segment: the text to translate
+    - src_lang (defaulted to be 'en'): the source language
+    - tgt_lang (defaulted to be 'fr'): the target language
```

Comment:
French should not be a default target. This is in conflict with the schema which has no default.

Reply:
Edited `src_lang` and `tgt_lang` description